### ISSUE TITLE
Squids are not affected by /mp

### ISSUE THREAD

**Issue by aggelosQQ** [TotalFreedom/TotalFreedomMod#1883]
Thread: http://totalfreedom.boards.net/thread/46093/squids-affected-mobpurge

Steven's Reply:
You are correct. The code for /mp includes all creatures, but squids are apparently not creatures according to Bukkit. Please create an issue at github.com/TotalFreedom/TotalFreedomMod/issues.

**Comment by lemonsked** [comment]
I think this would just need a `ent instanceof Squid` in the entity removing part.

**Comment by JeromSar** [comment]
@OxLemonxO You are correct, but this isn't a Bug. Bugs are issues that cause unintended behavior (i.e. functionality does not work or causes crashes).

**Comment by lemonsked** [comment]
Oh okay @JeromSar
